release-notes/SupportedFeatures-v1.0.0-rc2.md

```diff
@@ -141,7 +154,13 @@ Fill in with details.
 
 #### NetTcp  
 &ensp;&ensp; SetLingerOption isn't working correctly on OS X. This is only used on the Close(timeout) code path so the test itself is likely passing. This api required special handling on linux as this is a windows socket concept.  
```

Comment:
This issue with SetLingerOption has been resolved by a fix in CoreFx, so I think we can eliminate this and fix the entries for OS X related to NetTcp